docs/content/manual/harvester-rancher/03-manage-vm-downstream-harvester.md

```diff
@@ -21,3 +22,4 @@ Prerequisite: Harvester already imported to Rancher Dashboard
 1. Can `Stop`, `Restart` and `Migrate` virtual machine correctly
 1. Can `Clone` virtual machine correctly
 1. Can `Delete` virtual machine correctly
+1. Vm should be successfully created.
```

Comment:
I don't get this.
Is this the same to item 1~3?

> 1. Can create a single instance vm correctly
> 2. Can create multiple instances vm correctly
> 3. Can diaply all virtual machine information
> ...

Reply:
Exactly, why do have this at last check :P